README.md

```diff
@@ -147,6 +147,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
     - [Official](#official)
     - [Unofficial](#unofficial)
 - [Xcode](#xcode)
+    - [Extensions (Xcode 8+)](#extensions)
```

Comment:
`#extensions` would need to be changed to `#extensions-xcode-8`

Reply:
Thanks! Updated to #extensions-xcode-8 as you requested.